### PR TITLE
Fixes default location name after upgrade.

### LINKED ISSUE #77: rename the Default_Location to Default Location
Many end-to-end tests rely on finding Default Location , but our boxes have Default_location as they are upgraded from 6.1.z

### PR DESCRIPTION
**Pull request by san7ket**
Fixes #77 , tests failing as they are unable to find Default Location

`s/Default_Location/Default Location`

### CODE REVIEW

File: upgrade/helpers/tasks.py [hunk 1]
```diff
@@ -312,6 +312,10 @@ def post_upgrade_test_tasks(sat_host):
     execute(hammer, 'organization update --name "Default_Organization" '
             '--new-name "Default Organization" ',
             host=sat_host)
+    # Update the Default Location name, which was updated in 6.2
+    execute(hammer, 'organization update --name "Default_Location" '
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: Just add if to check if the TO_VERSION > 6.1 here

**Comment by san7ket** [comment]
@jyejare please re-visit

File: upgrade/helpers/tasks.py [hunk 2]
```diff
@@ -294,10 +294,11 @@ def sync_tools_repos_to_upgrade(client_os, hosts):
                 '1', tools_product, host)
 
 
-def post_upgrade_test_tasks(sat_host):
+def post_upgrade_test_tasks(sat_host, to_version):
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: You need not to have 'to_version' as a parameter as you can get that using os.environ.get('TO_VERSION') in any function.

So remove it as parameter and call as said above.

File: upgrade/helpers/tasks.py [hunk 3]
```diff
@@ -294,10 +294,11 @@ def sync_tools_repos_to_upgrade(client_os, hosts):
...
     """Run set of tasks for post upgrade tests
 
     :param string sat_host: Hostname to run the tasks on
+    :param string to_version: Upgraded satellite version
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: remove this as we are removing parameter.

File: upgrade/runner.py [hunk 4]
```diff
@@ -190,7 +190,7 @@ def product_upgrade(product):
                 execute(foreman_debug, 'satellite_{}'.format(sat_host),
                         host=sat_host)
                 # Execute tasks as post upgrade tier1 tests are dependent
-                post_upgrade_test_tasks(sat_host)
+                post_upgrade_test_tasks(sat_host, to_version)
```
Comment (jyejare) [inline upgrade/runner.py]: remove to_version as argument as we are removing param from func above

File: upgrade/helpers/tasks.py [hunk 5]
```diff
@@ -308,10 +308,15 @@ def post_upgrade_test_tasks(sat_host):
...
-    execute(hammer, 'organization update --name "Default_Organization" '
-            '--new-name "Default Organization" ',
-            host=sat_host)
+    if os.environ.get('TO_VERSION') not in ['6.0', '6.1']:
+        # Update the Default Organization name, which was updated in 6.2
+        execute(hammer, 'organization update --name "Default_Organization" '
+                '--new-name "Default Organization" ',
+                host=sat_host)
+        # Update the Default Location name, which was updated in 6.2
+        execute(hammer, 'organization update --name "Default_Location" '
```
Comment (evgeni) [inline upgrade/helpers/tasks.py]: this should be `location update`, right?

**Comment by sghai** [comment]
ACK

**Pull request merged**